pkgs/by-name/sq/sqld/package.nix

```diff
@@ -53,13 +61,25 @@ rustPlatform.buildRustPackage rec {
 
   env.ZSTD_SYS_USE_PKG_CONFIG = true;
 
+  # error[E0425]: cannot find function `consume_budget` in module `tokio::task`
+  env.RUSTFLAGS = "--cfg tokio_unstable";
+
   # requires a complex setup with podman for the end-to-end tests
   doCheck = false;
 
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
+      package = sqld;
+      command = "sqld --version";
+    };
```

Comment:
add versionCheckHook to nativeCheckInputs instead please

Reply:
Not `nativeCheckInstallInputs`?  https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook

Reply:
Yes, probably, sorry